### PR TITLE
sof-kernel-log-check: ignore device-mapper error for ehl

### PR DESCRIPTION
**Pull request by aiChaoSONG**
Fixes: #695

Signed-off-by: Chao Song <chao.song@linux.intel.com>

### CODE REVIEW

File: tools/sof-kernel-log-check.sh [hunk 1]
```diff
@@ -276,6 +276,9 @@ case "$platform" in
         # https://unix.stackexchange.com/questions/109294/mei-00000016-0-init-hw-failure
         ignore_str="$ignore_str"'|mei_me 0000:00:16\..: wait hw ready failed'
         ;;
+    ehl)
+        # https://github.com/thesofproject/sof-test/issues/695
+        ignore_str="$ignore_str"'|device-mapper: table: 253:0: thin-pool: unknown target type'
```
Comment (keqiaozhang) [inline tools/sof-kernel-log-check.sh:281]: Not sure if "253:0" will be changed in future, better use RegExp.

Comment (aiChaoSONG) [reply tools/sof-kernel-log-check.sh]: Updated

**Review by keqiaozhang — APPROVED** [review]
LGTM

**Review by marc-hb — CHANGES_REQUESTED** [review]
Why is docker running on this system?! Can we just `apt remove docker*` instead?

It's sometimes faster to look at the context of error messages rather than blindly adding an exclusion:

```
journalctl --boot | grep -C 5 thin

Jun 10 03:10:51 sh-ehl-rvp-i2s-09 dockerd[1014]: time="2021-06-10T03:10:51.572661404Z" level=info msg="scheme \"unix\" not registered, fallback to default scheme" module=grpc
Jun 10 03:10:51 sh-ehl-rvp-i2s-09 dockerd[1014]: time="2021-06-10T03:10:51.572691482Z" level=info msg="ccResolverWrapper: sending update to cc: {[{unix:///run/containerd/containerd.sock  <nil> 0 <nil>}] <nil> <nil>}" module=grpc
Jun 10 03:10:51 sh-ehl-rvp-i2s-09 dockerd[1014]: time="2021-06-10T03:10:51.572707458Z" level=info msg="ClientConn switching balancer to \"pick_first\"" module=grpc
Jun 10 03:10:51 sh-ehl-rvp-i2s-09 kernel: loop0: detected capacity change from 0 to 209715200
Jun 10 03:10:51 sh-ehl-rvp-i2s-09 kernel: loop1: detected capacity change from 0 to 4194304
Jun 10 03:10:51 sh-ehl-rvp-i2s-09 kernel: device-mapper: table: 253:0: thin-pool: unknown target type
Jun 10 03:10:51 sh-ehl-rvp-i2s-09 kernel: device-mapper: ioctl: error adding target to table
Jun 10 03:10:51 sh-ehl-rvp-i2s-09 dockerd[1014]: time="2021-06-10T03:10:51.684245814Z" level=error msg="[graphdriver] prior storage driver devicemapper failed: devicemapper: Error running deviceCreate (CreatePool) dm_task_run failed"
Jun 10 03:10:51 sh-ehl-rvp-i2s-09 dockerd[1014]: failed to start daemon: error initializing graphdriver: devicemapper: Error running deviceCreate (CreatePool) dm_task_run failed
```
```

**Comment by aiChaoSONG** [comment]
@marc-hb thank you, let's remove docker and close this.

**Pull request closed without merging**